products/qt.md

```diff
@@ -12,6 +12,13 @@ auto:
   # Upstream does not support filtering https://code.qt.io/qt/qt5.git
 -   git: https://github.com/qt/qt5.git
 releases:
+-   releaseCycle: "6.4"
+    eol: false # to be honest i couldnt find eol date for this release
+    # needs to fix after its eol annonced
```

Comment:
EoL for non LTS is release date plus 6 months -> `2022-03-30`, see 6.3 release and note at the end...

Reply:
you mean 2023 :D i just copy/paste without checking year :D

Reply:
@BiNZGi ^^

Reply:
Yes 2023 it is... :smile: